File: CompSci 1/Unit 12, Selection/RichterScale/RichterScale.java
import java.util.Scanner;

public class RichterScale {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a magnitude on the Richter scale --> ");
        double n = sc.nextDouble();
        System.out.print("\nDamage = ");

        if (n >= 8) {
            System.out.println("Most structures fall");
        }
        else if (n >= 7) {
            System.out.println("Most buildings destroyed");
        }
        else if (n >= 6) {
            System.out.println("Many buildings considerably damaged; some collapse");
        }
        else if (n >= 4.5) {
            System.out.println("Damage to poorly constructed buildings");
        }
        else if (n >= 3.5) {
            System.out.println("Felt by many people, no destruction");
        }
        else if (n >= 0) {
            System.out.println("Generally not felt by people");
        }
        else {
            System.out.println("Invalid number.");
        }
    }
}

File: CompSci 1/Unit 12, Selection/RichterScale/RichterScale.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Enter a magnitude on the Richter scale --> ")
    val n = sc.nextDouble()
    print("\nDamage = ")

    if (n >= 8) {
        println("Most structures fall")
    } else if (n >= 7) {
        println("Most buildings destroyed")
    } else if (n >= 6) {
        println("Many buildings considerably damaged; some collapse")
    } else if (n >= 4.5) {
        println("Damage to poorly constructed buildings")
    } else if (n >= 3.5) {
        println("Felt by many people, no destruction")
    } else if (n >= 0) {
        println("Generally not felt by people")
    } else {
        println("Invalid number.")
    }
}
